### ISSUE TITLE
Increase contrast of neighborhood text

### ISSUE THREAD

**Issue by cwood821** [Twin-Cities-Mutual-Aid/twin-cities-aid-distribution-locations#59]
Small change, the neighborhood name in sidebar list items currently fails accessibility contrast standards (AAA and AA). 

<img width="280" alt="Screen Shot 2020-06-03 at 8 55 05 PM" src="https://user-images.githubusercontent.com/7462489/83706416-034db080-a5dd-11ea-95bb-b430031fc8f9.png">
<img width="297" alt="Screen Shot 2020-06-03 at 8 55 10 PM" src="https://user-images.githubusercontent.com/7462489/83706417-03e64700-a5dd-11ea-94dd-51d3acc793df.png">

Screenshots are from Firefox Developer Edition dev tools, but you can confirm with https://webaim.org/resources/contrastchecker/

**Comment by jdalt** [comment]
I didn't even know you could formally validate contrast accessibility. This is cool.
